test/decl/protocol/special/coding/struct_codable_member_type_lookup.swift

```diff
@@ -174,7 +174,10 @@ struct SynthesizedStruct : Codable {
 
 // Structs which don't get synthesized Codable implementations should expose the
 // appropriate CodingKeys type.
-struct NonSynthesizedStruct : Codable {
+struct NonSynthesizedStruct : Codable { // expected-note{{'NonSynthesizedStruct' declared here}}
```

Comment:
Nit: I believe you can do this `// expected-note 4 {{'NonSynthesizedStruct' declared here}}` when the same note is expected to repeat.